data/mods/blazemod/blaze_blob.json

```diff
@@ -367,8 +399,8 @@
   },
   {
     "id": "gloople_act",
-    "type": "TOOL",
-    "category": "tools",
+    "type": "GENERIC",
```

Comment:
@mlangsdorf Sorry, but with those lines you broke the reverting item to 'gloople' after activation, so it will stay 'gloople_act' and spawn blobs as long as you want without any effort, and also it will stick to your hands forever if you try to wield it. Please fix it.